dev/global-settings.json

```
@@ -1,8 +1,7 @@
 {
   // This is set from an environment variable
   "experimentalFeatures": {
-    "showRepogroupHomepage": true,
-    "showEnterpriseHomePanels": true
+    "showRepogroupHomepage": true
```

Should this be `showEnterpriseHomepage`?
```suggestion
    "showEnterpriseHomepage": true
```

---

This is the current name of the feature flag. I think there are arguments to be made for renaming it (eg. removing the `enterprise` word from it as we start to use it more in Cloud), but it's out of scope for this PR.

---

Oh sorry I mistyped `showEnterpriseHomePanels` to `showEnterpriseHomepage`. Doesn't this PR want to enable `showEnterpriseHomePanels ` by default? Why are we enabling `showRepogroupHomepage` by default in dev and not `showEnterpriseHomePanels`?

---

This PR is switching the feature flag default for `showEnterpriseHomePanels` from `false` to `true`, so it doesn't need to be explicitly listed in `global-settings.json` anymore.

---

Ah so sorry 🤦